app/src/main/java/org/sopt/dateroad/domain/type/MonthType.kt

```diff
@@ -0,0 +1,20 @@
+package org.sopt.dateroad.domain.type
+
+enum class MonthType(val title: String) {
+    JANUARY("January"),
```

Comment:
```suggestion
    JANUARY(title = "January"),
```
이런 식으로 부탁합니다 그리고 얘네도 상수화 진행해주세요